packages/backend/server/src/schema.gql

```diff
@@ -67,14 +67,14 @@ type InviteUserType {
   """User avatar url"""
   avatarUrl: String
 
-  """User created date"""
-  createdAt: DateTime
+  """User email verified"""
+  createdAt: DateTime @deprecated(reason: "useless")
```

Comment:
it is statistically significant. For example, it can analyze the time distribution of users who have not completed the registration process, and the operation side will use